deploy/clowdapp.yaml

```diff
@@ -1,32 +1,44 @@
 ---
 apiVersion: v1
 kind: Template
+labels:
+  app: provisioning
+  template: provisioning
 metadata:
-  name: provisioning-backend
+  name: provisioning
+  annotations:
+    description: API backend for provisioning in console.redhat.com
+
 objects:
   - apiVersion: cloud.redhat.com/v1alpha1
     kind: ClowdApp
     metadata:
-      name: provisioning-backend
+      name: provisioning
+      labels:
+        service: provisioning
     spec:
       envName: ${ENV_NAME}
       deployments:
         - name: service
           minReplicas: ${{MIN_REPLICAS}}
-          web: true
+          webServices:
+            public:
+              enabled: true
+              apiPath: provisioning
           podSpec:
             image: ${IMAGE}:${IMAGE_TAG}
             initContainers:
               - name: run-migrations
+                image: "${IMAGE}:${IMAGE_TAG}"
                 command:
-                  - bash
+                  - sh
                   - -c
-                  - pbmigrate
+                  - /pbmigrate
```

Comment:
I haven't tried, but there is no reason to run our app in shell, can you simply run it directly? You will have to give it full path to the binary but that's easy to figure out I guess.

Reply:
nah, I just figured there is no bash so I changed to sh, but we might try running directly, it is probably the best :shrug:

Reply:
Just delete sh and -c and just run /pbmigrate. The same for the service itself - no need to have another process running in the container :)